content/en/synthetics/test_suites/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Test Suites
+description: Organize Synthetic Monitoring tests into logical groups.
+further_reading:
+- link: /synthetics/
+  tag: Documentation
+  text: Learn about Synthetic Monitoring
+- link: /monitors/types/synthetics/
+  tag: Documentation
+  text: Configure Synthetic test monitors
+- link: "/synthetics/troubleshooting/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Troubleshooting"
+---
+
+
+## Overview
+
+Synthetic Monitoring [Test Suites][1] let you organize multiple tests into a single collection for streamlined management, and troubleshooting. Group tests by user journey, environment, location, service, team, or any other dimension that fits your workflow. View aggregated results, identify failing components, and understand application performance across related tests, all from a unified view. 
```

Comment:
Recommend moving the link from the Overview to the "Create a test suite" section so it doesn't take the reader away from the page until they're actively doing something in the app.

```suggestion
Synthetic Monitoring Test Suites let you organize multiple tests into a single collection for streamlined management, and troubleshooting. Group tests by user journey, environment, location, service, team, or any other dimension that fits your workflow. View aggregated results, identify failing components, and understand application performance across related tests, all from a unified view. 
```